packages/driver/src/cypress/error_utils.js

```diff
@@ -431,7 +440,40 @@ const errorFromProjectRejectionEvent = (event) => {
   }
 
   // makeErrFromObj clones the error, so the original doesn't get mutated
-  return [makeErrFromObj(event.reason), event.promise]
+  return {
+    originalErr: event.reason,
+    err: makeErrFromObj(event.reason),
+    promise: event.promise,
+  }
+}
+
+const errorFromUncaughtEvent = (handlerType, event) => {
+  return handlerType === 'error' ?
+    errorFromErrorEvent(event) :
+    errorFromProjectRejectionEvent(event)
+}
+
+const logError = (Cypress, handlerType, err, handled = false) => {
+  Cypress.log({
+    message: err.message,
+    name: 'uncaught exception',
```

Comment:
So for unhandled rejections, this will still say 'uncaught exception'? Should it be switched to say `uncaught rejection`? Or is the intent to match the event name `uncaught:exception`?

Reply:
The way I see it, an exception is an exception, regardless of where it came from. The fact that it's from an unhandled promise rejection is an extra piece of information about the error, but doesn't define it. It's output to the console if the user cares to know. And yeah, it matches the `uncaught:exception` event.

Reply:
works for me :+1: